packages/params/src/index.ts

```diff
@@ -23,7 +25,7 @@ presetStatus.frozen = true;
  *
  * The active preset can be manually overridden with `setActivePreset`
  */
-export const ACTIVE_PRESET: PresetName =
+export const ACTIVE_PRESET =
```

Comment:
is this type now inferred to be `PresetName`?

Reply:
Apparently it's not needed, I can't think of a reason why it would be necessary. Not sure if it's a legacy forgotten though or an old uncommented reason